src/java.base/share/classes/java/util/LinkedList.java

```diff
@@ -1457,10 +1503,18 @@ public int lastIndexOf(Object o) {
             return rlist.lastIndexOf(o);
         }
 
+        public int findLastIndex(Predicate<? super E> filter) {
+            return rlist.findLastIndex(filter);
```

Comment:
Hmm, this view is supposed to be reversed... so last and first need a swap. This is a bug in the current codebase, a good bug to fix indeed.

Reply:
Ignore my fault, it's just delegating methods to 2 different, already reversed views...